900_building_testsuites/exercise-07/config/exercise-07_config_variant1.json

```diff
@@ -0,0 +1,31 @@
+//  Copyright 2020-2022 Robert Bosch GmbH
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//**************************************************************************
+
+{
+  "WelcomeString"   : "Hello... Robot Framework is running now!",
+
+  "Maximum_version" : "0.6.0",
```

Comment:
Hi Holger,
we assume that we are always compatible to old code when we make a new version,
therefore I would set the Maximum_version to 1.0.0 and increase Minimum_version to 0.6.0 for new features.
Thank you,
 Thomas

Reply:
Done